### ISSUE TITLE
[Question]: Joint training detail

### ISSUE THREAD

**Issue by cuifei659** [InternRobotics/InternNav#154]
### Question

Excellent work, thank you very much!

I noticed that in the inference demo you provided, the model's predicted trajectories are almost all within 0.5 meters. Why is this the case?

<img width="633" height="473" alt="Image" src="https://github.com/user-attachments/assets/216015bc-4a50-4d56-87bf-b0b929c67b5b" />

I have a few questions:

1、Does the loss calculation during joint training include both the action loss and the critic loss ?

2、Are the parameters used for joint training loss calculation (such as pred_digit, memory_digit, etc. for sampling the label trajectory) different from those used during S1 pre-training?

I look forward to your response!

**Comment by yuqiang-yang** [comment]
1. We provide all-zero depth for the model in `inference-only-demo`. So the trajectories are short. You should provide the correct depth image for the model in your implementation.
2. Currently, the joint training does not include the critic loss.
3. `pred_digit` is the same. `memory_digit` is different. S1 solely has 8 memory. But S1 of InternVLA-N1 only has 2 memory.

We will open-source the co-training code in two weeks.
